src/foraging_gui/Foraging.py

```diff
@@ -3278,18 +3254,15 @@ def _StartFIP(self):
         self.StartFIP.setChecked(False)
 
         if self.Teensy_COM == '':
-            logging.warning('No Teensy COM configured for this box, cannot start FIP workflow')
-            self.TeensyWarning.setText('No Teensy COM for this box')
-            self.TeensyWarning.setStyleSheet(self.default_warning_color)
```

Comment:
![image](https://github.com/user-attachments/assets/5479ae77-0d89-4ab8-af9d-cca14be1e06a)

Reply:
This photo of TeensyWarning should sufficiently depict what TeensyWarning looks likes, even though different text is assigned to it.